9x9-server/game.py

```diff
@@ -56,17 +56,14 @@ def join(self, client, room):
 
         if len(self.emptyRooms[0].clients) == 2:
             self.emptyRooms.pop(0)
-        # sendRandomState(client)
 
     def set(self, client, x, y):
         print(client.roomId)
-        room = self.rooms[client.roomId]
-        if not room.ready:
-            return
-
-        if room.clients[room.curMove] != client:
+        if client.roomId == -1:
+            client.send({
+                "message": "You're not conected to any room!"
```

Comment:
In the 9x9 protocol, there is a `msg`, not `message`.
And you can use `client.send('You are...', 'BAD')` instead because of client's `send()` method:
https://github.com/Pikne-Programy/9x9-server/blob/14a1dde0c14fa791f9435812da270a6d0d88151d/9x9-server/client.py#L68-L70